packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -68,10 +68,7 @@ export class SupportComponent {
 		await Promise.all( [
 			// Waits for all placeholder CSS elements to be removed from the DOM.
 			this.waitForQueryComplete(),
-			// Waits for one of the network request (triggered by the opening of the popover) to complete.
-			this.page.waitForResponse(
-				( response ) => response.status() === 200 && response.url().includes( 'kayako/mine?' )
-			),
+			this.page.waitForSelector( selectors.supportPopoverButton ),
```

Comment:
Yes, opening Inline Help no longer triggers any network request, except `block-editor/nux` and `me/settings`, but both are unnecessary re-fetches that can be elimitated by more optimized rendering.